subgraph/src/datasources/StakeAllocator.ts

```diff
@@ -34,7 +23,7 @@ export function handleAllocatedStake(event: AllocatedStakeEvent): void {
     const subjectType = event.params.subjectType;
     const subjectId = formatSubjectId(event.params.subject, event.params.subjectType);
 
-    if(subjectType === 2) {
+    if (subjectType === 2 || subjectType === 3) {
```

Comment:
Nice catch. I feel like we should add a util folder/file because a lot of these small methods/ logic are reused across different files